data/korath/korath weapons.txt

```diff
@@ -81,6 +139,30 @@ outfit "Korath Banisher"
 		"hit force" 40
 	description "The Korath like to keep their enemies close... but not too close."
 
+outfit "Korath Expeller"
+	category "Guns"
+	cost 819000
+	thumbnail "outfit/expeller"
+	"mass" 20
+	"outfit space" -20
+	"weapon capacity" -20
+	"gun ports" -1
+	weapon
+		sprite "projectile/expeller"
+			"frame rate" 60
+		sound "banisher"
+		"hit effect" "banisher impact"
+		"inaccuracy" .4
+		"velocity" 590
+		"lifetime" 1
+		"reload" 1
+		"firing energy" 1.75
+		"firing heat" 3.1
+		"shield damage" 1.4
+		"hull damage" .45
+		"hit force" 25
+	description "This low-power version of the Banisher is used on Korath fighters to push enemy vessels away from carriers and civilians."
```

Comment:
```suggestion
	description "This low-powered version of the Banisher is used on Korath fighters to push enemy vessels away from carriers and civilians."
```

Reply:
I dunno, "low-powered" sounds off to me

Reply:
Doesn't really matter either way, feel free to resolve if you like.